### ISSUE TITLE
Error: Unable to find electron-prebuilt's version number, either install it or specify an explicit version

### ISSUE THREAD

**Issue by shirishag75** [meetfranz/franz#1349]
I was trying to build franz locally and followed all the instructions - 

Just for completeness I did the following - 

```
$ sudo npm cache verify
[sudo] password for shirish: 
npm WARN npm npm does not support Node.js v10.15.2
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/
Cache verified and compressed (~/.npm/_cacache):
Content verified: 0 (0 bytes)
Index entries: 0
Finished in 0.026s
```

Then taking on the rebuild - 

```
$ npm run rebuild
npm WARN npm npm does not support Node.js v10.15.2
npm WARN npm You should probably upgrade to a newer version of node as we
npm WARN npm can't make any promises that npm will work with this version.
npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
npm WARN npm You can find the latest version at https://nodejs.org/

> franz@5.0.1-beta.1 rebuild /home/shirish/games/franz
> npx electron-rebuild

npx: installed 178 in 16.69s

An unhandled error occurred inside electron-rebuild
Unable to find electron-prebuilt's version number, either install it or specify an explicit version

Error: Unable to find electron-prebuilt's version number, either install it or specify an explicit version
    at Object.<anonymous> (/home/shirish/.npm/_npx/21040/lib/node_modules/electron-rebuild/lib/src/cli.js:81:19)
    at Generator.next (<anonymous>)
    at /home/shirish/.npm/_npx/21040/lib/node_modules/electron-rebuild/lib/src/cli.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/shirish/.npm/_npx/21040/lib/node_modules/electron-rebuild/lib/src/cli.js:4:12)
    at /home/shirish/.npm/_npx/21040/lib/node_modules/electron-rebuild/lib/src/cli.js:70:8
    at Object.<anonymous> (/home/shirish/.npm/_npx/21040/lib/node_modules/electron-rebuild/lib/src/cli.js:146:4)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 255
npm ERR! franz@5.0.1-beta.1 rebuild: `npx electron-rebuild`
npm ERR! Exit status 255
npm ERR! 
npm ERR! Failed at the franz@5.0.1-beta.1 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/shirish/.npm/_logs/2019-03-21T16_17_40_572Z-debug.log
```

the log - 

```
$ cat /home/shirish/.npm/_logs/2019-03-21T16_17_40_572Z-debug.log
0 info it worked if it ends with ok
1 warn npm npm does not support Node.js v10.15.2
2 warn npm You should probably upgrade to a newer version of node as we
3 warn npm can't make any promises that npm will work with this version.
4 warn npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
5 warn npm You can find the latest version at https://nodejs.org/
6 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'rebuild' ]
7 info using npm@5.8.0
8 info using node@v10.15.2
9 verbose run-script [ 'prerebuild', 'rebuild', 'postrebuild' ]
10 info lifecycle franz@5.0.1-beta.1~prerebuild: franz@5.0.1-beta.1
11 info lifecycle franz@5.0.1-beta.1~rebuild: franz@5.0.1-beta.1
12 verbose lifecycle franz@5.0.1-beta.1~rebuild: unsafe-perm in lifecycle true
13 verbose lifecycle franz@5.0.1-beta.1~rebuild: PATH: /usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/shirish/games/franz/node_modules/.bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
14 verbose lifecycle franz@5.0.1-beta.1~rebuild: CWD: /home/shirish/games/franz
15 silly lifecycle franz@5.0.1-beta.1~rebuild: Args: [ '-c', 'npx electron-rebuild' ]
16 silly lifecycle franz@5.0.1-beta.1~rebuild: Returned: code: 255  signal: null
17 info lifecycle franz@5.0.1-beta.1~rebuild: Failed to exec rebuild script
18 verbose stack Error: franz@5.0.1-beta.1 rebuild: `npx electron-rebuild`
18 verbose stack Exit status 255
18 verbose stack     at EventEmitter.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/index.js:283:16)
18 verbose stack     at EventEmitter.emit (events.js:189:13)
18 verbose stack     at ChildProcess.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
18 verbose stack     at ChildProcess.emit (events.js:189:13)
18 verbose stack     at maybeClose (internal/child_process.js:970:16)
18 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
19 verbose pkgid franz@5.0.1-beta.1
20 verbose cwd /home/shirish/games/franz
21 verbose Linux 4.19.0-2-amd64
22 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "rebuild"
23 verbose node v10.15.2
24 verbose npm  v5.8.0
25 error code ELIFECYCLE
26 error errno 255
27 error franz@5.0.1-beta.1 rebuild: `npx electron-rebuild`
27 error Exit status 255
28 error Failed at the franz@5.0.1-beta.1 rebuild script.
28 error This is probably not a problem with npm. There is likely additional logging output above.
29 verbose exit [ 255, true ]
```

So is the error is just too old npm or something else ?

**Desktop (please complete the following information):**
 - OS: [Debian Linux buster]
 - Browser [e.g. Firefox ]
 - Version [e.g. 60.5.1esr]

**Comment by djshadowxm82** [comment]
I've also ran into the same issue this afternoon on attempt to compile from source following the instructions verbatim as stated in the readme.md

**Comment by q4z1** [comment]
It's propably an issue with the version of npm and node. electron itself refers to yarn:
https://www.electron.build/#installation

So ... I used yarn running the following command `yarn add electron-builder --dev` (instead of `npm run rebuild`) in the Franz folder.
After succesfull build I was able to run `npm run dev` and `npm run start`

Regarding yarn:
I had to update my ubuntu nodejs to the newest (8.15.1) version.
(it might be a good idea to cleanup the node_modules folder before re-trying)
